pkgs/games/starruler2/data.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation {
+  pname = "starruler2-data";
+  version = "2022-08-30";
+
+  src = fetchFromGitHub {
+    owner = "OpenSRProject";
+    repo = "OpenStarRuler-Data";
+    rev = "681d87467f7e6bbdc3dddbbe4c6ab7a77fe0a781";
+    hash = "sha256-MaGPbfSv3/j98seR+p2mVOf2zJ4A6Fc+W03SesgnPpU=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    dir="$out/share/games/starruler2"
+    mkdir -p $dir
+    mv data locales maps mods scripts credits.txt sr2.icns sr2.ico "$dir"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A massive-scale 4X/RTS game set in space -- data files";
+    homepage = "https://github.com/OpenSRProject/OpenStarRuler-Data";
+    license = with licenses; [mit cc-by-nc-20]; # Art assets are licensed as CC-BY-NC 2.0.
```

Comment:
This is the package for the assets, and the assets' license is cc-by-nc-20. So why mit ?
Also I don't see the point of the comment, it's clear that this is the package for the assets, so if the license is just set to cc-by-nc-20 it's clear what the assets' license it

Reply:
This is the entire game, not just the art assets. It includes the game code, sound effects, etc. See the comment above about how the game is implemented as a mod.